src/nnet3bin/Makefile

```diff
@@ -11,13 +11,13 @@ BINFILES = nnet3-init nnet3-info nnet3-get-egs nnet3-copy-egs nnet3-subset-egs \
    nnet3-compute-from-egs nnet3-train nnet3-am-init nnet3-am-train-transitions \
    nnet3-am-adjust-priors nnet3-am-copy nnet3-compute-prob \
    nnet3-average nnet3-am-info nnet3-combine nnet3-latgen-faster \
-   nnet3-latgen-faster-parallel nnet3-show-progress nnet3-align-compiled \
-   nnet3-copy nnet3-get-egs-dense-targets nnet3-compute \
-   nnet3-discriminative-get-egs nnet3-discriminative-copy-egs \
-   nnet3-discriminative-merge-egs nnet3-discriminative-shuffle-egs \
-   nnet3-discriminative-compute-objf nnet3-discriminative-train \
-   discriminative-get-supervision nnet3-discriminative-subset-egs \
-   nnet3-discriminative-compute-from-egs
+   nnet3-copy nnet3-show-progress nnet3-align-compiled \
+   nnet3-get-egs-dense-targets nnet3-compute nnet3-modify-learning-rates \
```

Comment:
I think nnet3-modify-learning-rates has been removed from master.
Just undo all the changes that you did here, and just add the nnet3-copy-multiple-egs at the end since that is the only change you are making here.